nasl/nasl_http.c

```diff
@@ -123,7 +124,13 @@ _http_req (lex_ctxt * lexic, char *keyword)
       if (!ua || strlen (g_strstrip (ua)) == 0)
         {
           g_free (ua);
-          ua = g_strdup ("Mozilla/5.0 [en] (X11, U; OpenVAS)");
+          vendor = g_strdup (vendor_version_get ());
+          if (!vendor || *vendor == '\0')
+            ua = g_strdup_printf ("Mozilla/5.0 [en] (X11, U; OpenVAS-VT %s)",
+                                         OPENVAS_NASL_VERSION);
```

Comment:
Code formatting while at it. OPENVAS_NASL_VERSION should start under "Mozilla...